registry/quilt_server/views.py

```diff
@@ -164,15 +146,7 @@ def _validate_username(username):
 
 @app.route('/login')
```

Comment:
Should be `/api/login`. Same for other API routes.

(We already have a `/login` that's a redirect to a normal page.)

Reply:
The current `/login` is actually served by Flask directly. What routes should/shouldn't be `/api`? I think `/api` should be about interacting with Quilt features, like packages and stuff, and auth would be in a different namespace. The top-level namespace is fine for now, and we can change it later if necessary. I think we should probably move some stuff out of `/api`, like `/api/users`, but that's a separate PR.